robottelo/constants.py

```diff
@@ -837,7 +837,7 @@
         'skip_for_ui': True},
     {'name': 'HostCollection', 'controller': 'katello_host_collections'},
     {'name': 'Architecture', 'controller': 'architectures'},
-    {'name': 'HardwareModel', 'controller': 'models'},
+    {'name': 'HardwareModel', 'controller': 'models', 'skip_for_ui': True},
```

Comment:
@rplevka I see that hardware model page is empty by default, so search button is not present.  Is this a bug?